### PR TITLE
modify tests to case-match the 'expected' file names to actual file names #2824 

### PR DESCRIPTION
**Pull request by ZiXian92**
For #2824

### CODE REVIEW

File: src/main/webapp/WEB-INF/classes/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/META-INF
```
Comment (damithc) [inline src/main/webapp/WEB-INF/classes/.gitignore]: Unstage all changes not relevant to this issue.

**Comment by damithc** [comment]
@ZiXian92 always look through the changeset to ensure no irrelevant changes are included. Also set the assignee and status labels correctly, as mentioned in process.md

**Comment by ZiXian92** [comment]
Regarding assigning myself to the pull request, I was unable to assign anyone nor set the labels somehow. Regarding removing irrelevant commits, do I include merges from the main repository? Also, after removing irrelevant commits, do I close this pull request and create another one?

**Comment by damithc** [comment]
I guess GitHub doesn't allow you to assign yourself. In that case, we'll do
it ourselves. Not to worry.

**Pull request closed without merging**